server/controller/src/main/java/ai/starwhale/mlops/domain/evaluation/EvaluationService.java

```diff
@@ -88,10 +88,9 @@ public List<AttributeVo> listAttributeVo() {
     }
 
     public ConfigVo getViewConfig(ConfigQuery configQuery) {
-        Long userId = userService.currentUserDetail().getId();
         Long projectId = projectService.getProjectId(configQuery.getProjectUrl());
 
-        ViewConfigEntity viewConfig = viewConfigMapper.findViewConfig(userId, projectId, configQuery.getName());
+        ViewConfigEntity viewConfig = viewConfigMapper.findViewConfig(projectId, configQuery.getName());
```

Comment:
Should we change the create api's role from "'OWNER', 'MAINTAINER', 'GUEST'"  to "'OWNER', 'MAINTAINER'"?
The code is here: 
`@Operation(summary = "Create or Update View Config")
    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "ok")})
    @PostMapping(
            value = "/project/{projectUrl}/evaluation/view/config",
            produces = MediaType.APPLICATION_JSON_VALUE)
    @PreAuthorize("hasAnyRole('OWNER', 'MAINTAINER', 'GUEST')")
    ResponseEntity<ResponseMessage<String>> createViewConfig(
            @Valid @PathVariable("projectUrl") String projectUrl,
            @Valid @RequestBody ConfigRequest configRequest
    );`

Reply:
Done